R/Data-validity.R

```diff
@@ -1,35 +1,120 @@
-#' Internal Helper Functions for Validation of `GeneralData` Objects.
+#' Internal Helper Functions for Validation of `GeneralData` and `Data` Objects.
```

Comment:
I would not try to list all classes here. otherwise title becomes too long with the other 5 classes.